components/merkledb/src/views/address.rs

```diff
@@ -0,0 +1,207 @@
+use std::{borrow::Cow, num::NonZeroU64};
+
+use crate::BinaryKey;
+
+pub fn key_bytes<K: BinaryKey + ?Sized>(key: &K) -> Vec<u8> {
+    concat_keys!(key)
+}
+
+/// Represents the address of an index in the database.
+///
+/// An address has a string *name* and an optional byte *key*. An index is uniquely identified
+/// by its address. Different addresses correspond to different indexes. Addresses with the same
+/// name and differing prefixes are said to belong to the same *group* (see also [`Group`]). Groups
```

Comment:
I suppose that prefix in this context a little bit misleading. What it means? Is this an address suffix or prefix for index key?